palera1n.sh

```diff
@@ -158,11 +159,11 @@ _exit_handler() {
     [ $? -eq 0 ] && exit
     echo "[-] An error occurred"
 
-    cd logs
```

Comment:
script might not be at the root dir here, so this won't always point to the actual `logs` dir